openedx_events/enterprise/signals.py

```diff
@@ -8,16 +8,19 @@
 docs/decisions/0003-events-payload.rst
 """
 
-from openedx_events.enterprise.data import LicenseLifecycle
+from openedx_events.enterprise.data import SubscriptionLicenseData
 from openedx_events.tooling import OpenEdxPublicSignal
 
 # .. event_type: org.openedx.enterprise.subscription.license.modified.v1
-# .. event_name: LICENSE_CREATED
-# .. event_description: emitted with Subscription.License is created.
-# .. event_data: LicenseLifecycle
-LICENSE_CREATED = OpenEdxPublicSignal(
-    event_type="org.openedx.enterprise.subscription.license.created.v1",
+# .. event_name: LICENSE_MODIFIED
+# .. event_description: emitted  Subscription License is modified.
+# .. event_data: SubscriptionLicenseData
+# WARNING: This event is being used for prototype purposes and should not be used, and should not be copied as a fully vetted pattern.
+# TODO(EventBus): Figure out if the follow signal needs to be split into many.
+#                 Probably will need ton of enterprise input.
```

Comment:
1. I moved as part of the description so that automated docs pick up the warning and todo.
2. I adjust the todo comment a bit. Thoughts?
3. I had a different format for `TODO(EventBus):`, but I like yours better. Since I imagine this is the first use, I will update and inform the others.
```suggestion
# .. event_description: emitted  Subscription License is modified.
#     WARNING: This event is being used for event bus prototype purposes and
#          should not be used, and should not be copied as a fully vetted pattern.
#     TODO(EventBus): Delete or make production ready post-prototype phase.
# .. event_data: SubscriptionLicenseData
```